api/room/schema.py

```diff
@@ -260,6 +262,7 @@ class Mutation(graphene.ObjectType):
             of the calendar\n- office_id: The unique identifier of the office \
             where the room is found[required]\n- cancellation_duration: \
                 The cancellation duration of a booking\
+            \n- structure_id: The office structure id related to the room\
```

Comment:
According to your task description, a relationship is to be established between the `office_structures` and the `rooms`. In this implementation, the `structure_id` is not being validated to ensure that there's an `office_structure` with that particular `structure_id` - which means no relationship between the two will always be there. Kindly check into this.